python/composio/local_tools/local_workspace/cmd_manager/actions/edit_cmd.py

```diff
@@ -54,18 +56,15 @@ class EditFile(BaseAction):
     def execute(
         self, request_data: EditFileRequest, authorisation_data: dict
     ) -> BaseResponse:
-        self._setup(request_data)
+        workspace = workspace_utils.get_workspace_by_id(self.workspace_id)
```

Comment:
This will be a common code across actions.
Maybe create some utility around it. Maybe a base class?